src/Core/Settings.cpp

```diff
@@ -4872,6 +4872,9 @@ Limit on size of a single batch of file segments that a read buffer can request
 )", 0) \
     DECLARE(UInt64, filesystem_cache_reserve_space_wait_lock_timeout_milliseconds, 1000, R"(
 Wait time to lock cache for space reservation in filesystem cache
+)", 0) \
+    DECLARE(Bool, filesystem_cache_prefer_bigger_buffer_size, true, R"(
+Prefer bigger buffer size if filesystem cache is enabled to avoid writing small file segments which detiriorate cache performance
```

Comment:
```suggestion
Prefer bigger buffer size if filesystem cache is enabled to avoid writing small file segments which deteriorate cache performance
```

Reply:
i think it also makes sense to mention that on the flip side it may increase memory usage